pmultiqc/modules/quantms/quantms.py

```
@@ -4265,6 +4447,229 @@ def draw_msms_scans_top_n(self, top_n_data):
+                .reset_index(drop=True)
+            )
+            cont_df["contaminant_percent"] = (cont_df["average_intensity"] / group["average_intensity"].sum()) * 100
+
+            plot_dict[file_name] = dict(zip(cont_df["cont_accession"], cont_df["contaminant_percent"]))
+            plot_cats.extend(cont_df["cont_accession"].tolist())
+
+        plot_cats = list(set(plot_cats))
+        if "Other" in plot_cats:
+            plot_cats_ordered = [x for x in plot_cats if x != "Other"] + [x for x in plot_cats if x == "Other"]
+
+        result_dict = dict()
```

Pylint (3.3.7)

[refactor] Consider using '{}' instead of a call to 'dict'.

(R1735)